source/includes/fact-mongosync-balancer.rst

```diff
@@ -0,0 +1,7 @@
+.. important::
+
+   When the source or destination cluster is a sharded cluster, stop the
+   balancer on both clusters and don't run the :dbcommand:`moveChunk` or
+   :dbcommand:`moveRange` commands for the entire lifetime of the
```

Comment:
[suggestion]

Consider using slightly stronger language: "When the source or destination cluster is a sharded cluster, you must stop the balancer on both clusters and not run any `moveChunk` or `moveRange` commands for the duration of the migration."

Reply:
Updating!